### PR TITLE
[REVIEW] Adding faiss conda package

### PR DESCRIPTION
**Pull request by cjnolet**

### CODE REVIEW

**Comment by ajschmidt8** [comment]
rerun tests

**Comment by ajschmidt8** [comment]
@cjnolet, it looks like `conda` is having some issues with the package version syntax here. mind checking that out?

https://gpuci.gpuopenanalytics.com/job/rapidsai/job/integration/job/prb/job/env-pkg-nightly-build/123/CONDA_CONFIG_FILE=conda%2Frecipes%2Fversions.yaml,CONDA_USERNAME=rapidsai-nightly,CUDA_VER=10.2,DEFAULT_CUDA_VER=10.1,PYTHON_VER=3.7,RAPIDS_VER=0.15/console

image

File: conda/recipes/rapids-build-env/meta.yaml [hunk 1]
```diff
@@ -62,6 +62,8 @@ requirements:
     - distributed {{ distributed_version }}
     - dlpack {{ dlpack_version }}
     - double-conversion {{ double_conversion_version }}
+    - faiss-proc=*=cuda {{ faiss_version }}
```
Comment (kkraus14) [inline conda/recipes/rapids-build-env/meta.yaml:65]: Typically these `proc` packages just use version `1.0.0`so you don't want to version them

Comment (cjnolet) [reply conda/recipes/rapids-build-env/meta.yaml:65]: I've removed the faiss version

**Comment by cjnolet** [comment]
@ajschmidt8 Awesome. It looks like CI passed

**Pull request merged**